meta/Meta.cpp

```diff
@@ -6954,7 +6976,8 @@ void Meta::meta_sai_on_fdb_flush_event_consolidated(
             continue;
         }
 
-        if (fdbTypeAttr->getSaiAttr()->value.s32 != type->value.s32)
+        if ((type->value.s32 != SAI_FDB_FLUSH_ENTRY_TYPE_ALL) &&
+            (fdbTypeAttr->getSaiAttr()->value.s32 != type->value.s32))
```

Comment:
Aren't these values from different enum types? one from ..fdb_flush_attr_type.. and another one from ...fdb_attr_type..?

Reply:
May be we want this?
```
sai_attribute_t *fdb_type_attr_p = fdbTypeAttr->getSaiAttr();
        if ((flush_type_attr_p->value.s32 == SAI_FDB_FLUSH_ENTRY_TYPE_DYNAMIC &&
             fdb_type_attr_p->value.s32 != SAI_FDB_ENTRY_TYPE_DYNAMIC) ||
             (flush_type_attr_p->value.s32 == SAI_FDB_FLUSH_ENTRY_TYPE_STATIC &&
             fdb_type_attr_p->value.s32 != SAI_FDB_ENTRY_TYPE_STATIC))
        {
            // entry type is not matching on this fdb entry
            continue;
        }
```

Reply:
Also at line number 6962
Instead of this
```
SWSS_LOG_NOTICE("processing consolidated fdb flush event of type: %s",
            sai_metadata_get_fdb_entry_type_name((sai_fdb_entry_type_t)type->value.s32));
```
shall we have
```
SWSS_LOG_NOTICE("processing consolidated fdb flush event of type: %s",
            sai_metadata_get_fdb_flush_entry_type_name((sai_fdb_flush_entry_type_t)type->value.s32));
```

Reply:
yes you are right, those are different entry types, it happens to work, since ALL has different enum value than STATIC and DYNAMIC, but it works here :P, i will update that